test_pull_requests

```diff
@@ -948,15 +1013,39 @@ popd
                   break
                 end
               end
+              # the main pull request and all of the co-requisite pull requests haven't been
+              # updated since the last trusted trigger, so we can begin testing
               update_evaluated_markers(repo_to_pull_request, trigger_updated_at, settings)
               submit_test_job = true
             else
+              # no trusted trigger statement has been made, so we cannot build or test this pull
+              # TODO: only pull requests with trusted triggers get process_pull_request called
```

Comment:
there's probably some nuance I'm missing, but why, exactly, can't we replace [THIS](https://github.com/stevekuznetsov/test-pull-requests/blob/master/test_pull_requests#L1174-L1203) block with `trusted_trigger_time`? We seem to update with the `ACTION_NOT_TEAM` comment right then and there if we don't find a trusted trigger